### PR TITLE
Update chanstats.example.conf

### PR DESCRIPTION
**Pull request by SketchNI**
Added in extra (optional) smileys.

### CODE REVIEW

**Comment by SaberUK** [comment]
Is there any reason you didn't just add it to the existing smiley list? If someone doesn't want them then they can be removed easily.

**Comment by SketchNI** [comment]
It'll save time (seconds, really) for people that don't want them.

File: data/chanstats.example.conf [hunk 1]
```diff
@@ -20,9 +20,22 @@ module
 	 */
 	prefix = "anope_"
 
+	/*
+	 * Default smileys. These come with Anope.
+	 * Comment these out before uncommeting the extended smileys.
```
Comment (GLolol) [inline data/chanstats.example.conf]: uncomme**n**ting, not uncommeting

**Comment by Adam-** [comment]
This is ridiculous. Add them to the default and perhaps.

**Comment by Adam-** [comment]
Reopen if you want to change this.

**Pull request closed without merging**